databases/week2/part4.sql

```diff
@@ -0,0 +1,41 @@
+CREATE DATABASE Book_Library;
+USE Book_Library;
+
+CREATE TABLE
+    `Book_Library`.`author` (
+        `author_id` INT NOT NULL,
```

Comment:
In line 6 why `author_id` instead of just `id`? or why different naming style compared to other  tables? table book, why not book_id  ?

Reply:
I had some troubles with fk so this made better logic in my head but i fixed it now.